sdk/cosmosdb/cosmos/.vscode/settings.json

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
This file is not related to debugging, should keep it?

Reply:
right. However, I suggest to delete all `settings.json` too because they are opinionated and I am not sure why they are checked in in the first place.

Reply:
There are some useful ones like format on save, double  quotes, etc. It would be ideal if some centralized settings would work.  Unfortunately I don't think `code sdk/storage/storage-blob` would load vs code settings from repo root folder.

Reply:
I still think they are opinionated and should not be tracked by git. If I wanted to have my own custom `.vscode/settings.json`, it could be accidentally checked into master.

Reply:
Some of them actually help maintain our convention for the repo (defined in sdk/.prettierrc.json).  Which ones do you think should not be there? Maybe just remove those?

If removing it for consistency (most of our libraries don't have this file) then I am fine.  I will have to rely on the `format` npm script more.

Reply:
My opinion on which config should be kept is irrelevant because each one has their own thing. This change should not discourage you from having your own `settings.json`, it just makes sure it is not tracked by git. As for consistency, I imagine some of us do not even use vscode. A better approach to maintain consistency is to have git hooks for running linters/formatters.

Reply:
My point is some settings there are standard, not opinionated. You are right, git hook to format is a good approach.

Reply:
Done, please take a look: https://github.com/Azure/azure-sdk-for-js/pull/10219

Reply:
I believe this has been resolved now that #10219 is about to be merged.